docs/core/build/cross-space/crc20-cross-space.md

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 1
+title: CRC20 Tokens Cross Space
+displayed_sidebar: coreSidebar
+---
+
+This tutorial guides developers on transferring any CRC20 token from Core Space to eSpace, using [FansCoin (FC)](https://confluxscan.io/token/cfx:achc8nxj7r451c223m18w2dwjnmhkd6rxawrvkvsy2) as an example. The goal is to transfer a CRC20 token (A) issued on coreSpace to become an ERC20 token (eA) in eSpace. Currently, if there is no corresponding token in eSpace, it's not possible to use the official [cross-space bridge](https://confluxhub.io/espace-bridge/cross-space) directly.
```

Comment:
Using FC as example seems improper here as the main content of this article is now how to enable token transfer.